src/VisualStudio/Core/Def/Implementation/Progression/GraphQueries/InheritedByGraphQuery.cs

```diff
@@ -32,7 +27,7 @@ internal sealed class InheritedByGraphQuery : IGraphQuery
                 if (namedType.TypeKind == TypeKind.Class)
                 {
                     var derivedTypes = await namedType.FindDerivedClassesAsync(solution, cancellationToken).ConfigureAwait(false);
```

Comment:
Doesn't this does a bunch of extra work to go and find all the levels of inheritance, only to ignore it after you have it?  Consider making `FindDerivedClassesAsync` able to return just one level directly.